python/plugins/processing/algs/qgis/PointsToPaths.py

```diff
@@ -48,18 +48,19 @@
 
 class PointsToPaths(QgisAlgorithm):
     INPUT = 'INPUT'
+    CLOSE_PATH = 'CLOSE_PATH'
     GROUP_FIELD = 'GROUP_FIELD'
     ORDER_FIELD = 'ORDER_FIELD'
     DATE_FORMAT = 'DATE_FORMAT'
     OUTPUT = 'OUTPUT'
     OUTPUT_TEXT_DIR = 'OUTPUT_TEXT_DIR'
-
+    
```

Comment:
```suggestion

```

I think these spaces will need to be removed as well to make the linter checks happy.